vendors/circom/.bazelrc

```diff
@@ -15,6 +15,18 @@ build:macos_arm64 --config=macos
 build:macos_arm64 --cpu=darwin_arm64
 build:macos_arm64 --host_cpu=darwin_arm64
 
+# This config refers to building sources with nvcc.
```

Comment:
```suggestion
# Use this config to build sources with nvcc.

Reply:
Actually this comment comes from https://github.com/tensorflow/tensorflow/blob/fbf63f935c2977333103703b5fa769bf6b831635/.bazelrc#L224, also there are [3 places](https://github.com/kroma-network/tachyon/blob/a74a3fff8fd9812a99bd939d267a3760ec645214/.bazelrc#L34-L50) that use this kind of patterns, do you think it's better to modify?

Reply:
I thought it was strange initially, but looking back it seems fine~ I'll resolve